dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/k8s.vue

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
The UI part need to change in the ui-next.

Reply:
Changed

Reply:
Done